Patches/AnnouncementPatch.cs

```diff
@@ -72,7 +73,8 @@ public Announcement ToAnnouncement()
         return result;
     }
     public static List<ModNews> AllModNews = [];
-    public static string ModNewsURL = "https://raw.githubusercontent.com/0xDrMoe/TownofHost-Enhanced/main/Resources/Announcements/modNews-";
+    public static string ModNewsURL = "https://raw.githubusercontent.com/0xDrMoe/TownofHost-Enhanced/AnnouncementFor160/Resources/Announcements/modNews-";
```

Comment:
if you change the url, announcement won't be real time, they will have to wait for next release to get the announcement

Reply:
I know, I'm doing this to test